.rhcicd/clowdapp-backend.yaml

```
@@ -166,6 +217,25 @@ objects:
                   secretKeyRef:
                     name: notifications-backend-db
                     key: ${DB_SECRET_PASSWORD_KEY}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: floorplan
+  data:
+    floorplan.yaml: |
+      - prefix: insights/notifications/splunk
+        query: >-
+          SELECT 
+            DISTINCT "account_id" 
+            FROM "endpoints"
+            WHERE "endpoint_sub_type"='splunk';
```

Is worth to lock the endpoint_type to 3 (camel)
```suggestion
            WHERE "endpoint_type"=3 AND "endpoint_sub_type"='splunk';
```